Dependency collection script fails during `pnpm version`

**What happened**: The `collect-dependencies.js` script fails during `preversion`.

**What should have happened**: The `collect-dependencies.js` script should work as expected when running `pnpm version`, just like `node tools/collect-dependencies.js -- -o src/api/dependency-discovery/deps.txt` works.

**How to reproduce it (as precise as possible)**: You'd need to remove the `postversion` script in `package.json` to avoid uploading the new commit to `telescope/master`. After that, you can run `pnpm version`.

**Anything else we need to know?**: This error log:

```
> @senecacdot/telescope@2.8.1 preversion /home/gerardo/workspace/telescope
> node tools/collect-dependencies.js -- -o src/api/dependency-discovery/deps.txt && git add src/api/dependency-discovery/deps.txt

Starting to collect dependencies
Invoking pnpm list...
Processing file...
/home/gerardo/workspace/telescope/tools/collect-dependencies.js:28
  ...new Set([...output.matchAll(regex)].map((results) => results[1].replaceAll('+', '/')).sort()),
                                                                     ^

TypeError: results[1].replaceAll is not a function
    at /home/gerardo/workspace/telescope/tools/collect-dependencies.js:28:70
    at Array.map (<anonymous>)
    at Object.<anonymous> (/home/gerardo/workspace/telescope/tools/collect-dependencies.js:28:42)
    at Module._compile (internal/modules/cjs/loader.js:1085:14)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1114:10)
    at Module.load (internal/modules/cjs/loader.js:950:32)
    at Function.Module._load (internal/modules/cjs/loader.js:790:12)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:75:12)
    at internal/main/run_main_module.js:17:47
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @senecacdot/telescope@2.8.1 preversion: `node tools/collect-dependencies.js -- -o src/api/dependency-discovery/deps.txt && git add src/api/dependency-discovery/deps.txt`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the @senecacdot/telescope@2.8.1 preversion script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/gerardo/.npm/_logs/2022-03-11T21_40_28_436Z-debug.log
```

--- a/tools/collect-dependencies.js
+++ b/tools/collect-dependencies.js
@@ -25,7 +25,7 @@ const regex = /.*\.pnpm\/(@?[^@]+).*/g;
 
 console.log('Processing file...');
 const packageNames = [
-  ...new Set([...output.matchAll(regex)].map((results) => results[1].replaceAll('+', '/')).sort()),
+  ...new Set([...output.matchAll(regex)].map((results) => results[1].replace('+', '/')).sort()),
 ];
 
 const buffer = packageNames.join('\n');

